measurements/attribute_based.py

```diff
@@ -62,56 +62,59 @@ def size_and_distance(dataloader, args):
             detect_info = {}
     elif FACE_DETECT == 1:
         cascPath = "util_files/haarcascade_frontalface_default.xml"
-        faceCascade = cv2.CascadeClassifier(cv2.data.haarcascades + cascPath)
-
+        #faceCascade = cv2.CascadeClassifier(cv2.data.haarcascades + cascPath)
+        faceCascade = cv2.CascadeClassifier(cascPath)
```

Comment:
why is this change here? if you need it locally, some environments don't, so please include a try/catch statement that captures both, gender_based.py may have something similar right now